src/ophyd_async/core/_flyer.py

```diff
@@ -29,7 +27,7 @@ async def stop(self):
         """Stop flying and wait everything to be stopped"""
 
 
-class HardwareTriggeredFlyable(
+class StandardFlyer(
```

Comment:
I'm pretty sure `TriggerLogic` is now the wrong name, but can't think of a better one immediately so let's leave it for now